tests/unit_tests/data_steward/tools/load_vocab_test.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+
+import mock
+from google.cloud.bigquery import Dataset, DatasetReference, Client
+from google.cloud.storage import Blob
+
+import common
+from tools import load_vocab
+
+
+class LoadVocabTest(unittest.TestCase):
```

Comment:
Another main function `load` is not tested

Reply:
You're right- this should be acceptance criteria for the tickets created